src/client/client.go

```diff
@@ -63,9 +67,10 @@ func NewMetricsClientFromAddress(addr string, metrics bool, prefix string) (*API
 }
 
 // NewFromAddress constructs a new APIClient for the server at addr.
-func NewFromAddress(addr string) (*APIClient, error) {
+func NewFromAddress(addr string, maxConcurrentStreams uint) (*APIClient, error) {
```

Comment:
Changing this signature seems like it's likely to break some existing user code. Could we use default here and have a different method for creating one with a non default number of streams?